### PR TITLE
[opt](nereids)clear min/max column stats if table is partially analyzed

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
if user queries newly loaded data (the new data are not analyzed), optimizer may generate inefficient plan because the newly loaded data is out of column stats min-max range.

In this pr, we will ignore min-max if there are newly loaded data.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Review by Jibing-Li — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java [hunk 1]
```diff
@@ -773,6 +777,10 @@ private Statistics computeCatalogRelation(CatalogRelation catalogRelation) {
                 idxId = olapScan.getSelectedIndexId();
             }
         }
+        if (deltaRowCount > 0) {
+            LOG.info(catalogRelation.getTable().getName()
+                    + " is partially analyzed, clear min/max values in column stats");
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java:782]:
```
if (LOG.isDebugEnabled()) {
  LOG.debug("{} is partially analyzed, clear min/max values in column stats", catalogRelation.getTable().getName());
}
```

**Review by morrySnow — COMMENTED** [review]
could we add some test?

**Comment by englefly** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by Jibing-Li — APPROVED** [review]

**Pull request merged**